chrome/locale/en-US/zotero/reader.ftl

```diff
@@ -0,0 +1,66 @@
+pdfReader-underlineText = Underline Text
+pdfReader-highlightText = Highlight Text
+pdfReader-addText = Add Text
+pdfReader-selectArea = Select Area
+pdfReader-draw = Draw
+
+pdfReader-highlightAnnotation = Highlight annotation
+pdfReader-underlineAnnotation = Underline annotation
+pdfReader-noteAnnotation = Note Annotation
+pdfReader-textAnnotation = Text Annotation
+pdfReader-imageAnnotation = Image Annotation
+
+pdfReader-find-in-document = Find in Document
+
+pdfReader-a11yMoveAnnotation = Use the arrow keys to move the annotation.
+pdfReader-a11yEditTextAnnotation = To move the end of the text annotation, hold { general-key-shift } and use the left/right arrow keys. To move the start of the annotation, hold { general-key-shift }-{ option-or-alt } and use the arrow keys.
+pdfReader-a11yResizeAnnotation = To resize the annotation, hold { general-key-shift } and use the arrow keys.
+pdfReader-a11yAnnotationPopupAppeared = Use Tab to navigate the annotation popup.
+
+pdfReader-a11yAnnotationCreated = { $type ->
+    [highlight] { pdfReader-highlightAnnotation }
+    [underline] { pdfReader-underlineAnnotation }
+    [note] { pdfReader-noteAnnotation }
+    [text] { pdfReader-textAnnotation }
+    [image] { pdfReader-imageAnnotation }
+   *[other] Annotation
+} created.
+
+pdfReader-a11yAnnotationSelected = { $type ->
+    [highlight] { pdfReader-highlightAnnotation } selected. { pdfReader-a11yEditTextAnnotation }
+    [underline] { pdfReader-underlineAnnotation } selected. { pdfReader-a11yEditTextAnnotation }
+    [note] { pdfReader-noteAnnotation } selected. { pdfReader-a11yMoveAnnotation }
+    [text] { pdfReader-textAnnotation } selected. { pdfReader-a11yMoveAnnotation } { pdfReader-a11yResizeAnnotation }
+    [image] { pdfReader-imageAnnotation } selected. { pdfReader-a11yMoveAnnotation } { pdfReader-a11yResizeAnnotation }
+   *[other] Annotation selected.
+} { $popupVisible -> 
+    [yes] { pdfReader-a11yAnnotationPopupAppeared }
+    *[other] { "" }
+}
```

Comment:
Since this shows up on Transifex, it's not ideal moving this much logic into Fluent. It might be necessary for a visual string where different languages have different conventions around sentence separators, but since this is only for screen readers anyway, that doesn't really matter — a period and a space should be sufficient. So if possible, probably better to handle most of this in JS and just have the "[ annotation ] selected." part here.

And now that we have placeholders where the order can be changed, we could probably try to just have a single string with `{ $annotation } selected.`, pass in the appropriate Fluent string (`pdfReader-textAnnotation`, etc.) as `$annotation`, and see if anyone complains that that doesn't work.